tests/SpecTests/Operation.php

```diff
@@ -568,9 +559,7 @@ private function executeForSession(Session $session, FunctionalTestCase $test, C
 
             case 'withTransaction':
                 /** @var self[] $callbackOperations */
-                $callbackOperations = array_map(function ($operation) {
-                    return self::fromConvenientTransactions($operation);
-                }, $this->arguments['callback']->operations);
+                $callbackOperations = array_map(fn ($operation) => self::fromConvenientTransactions($operation), $this->arguments['callback']->operations);
```

Comment:
Do you want to split the `array_map()` parameters across multiple lines for readability? This extends past 120 characters.